task-alphabet/html.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Alpabet LLC</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="stylesheet" href="desctop.css" media="(min-width: 700px)">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+</head>
+<body>
+    <div class="main-box">
+        <input type="checkbox" class="hidden-menu" name="hide">
+        <aside class="menu">
+            <h1 class="header">Alphabet LLC</h1>
+            <nav class="news">
+                <div class="box-options">
+                    <ul class="options">
+                        <li class="option"><a href="#0">News</a></li>
+                        <li class="option"><a href="#0">Articles</a></li>
+                        <li class="option"><a href="#0">Bookmarks</a></li>
+                        <li class="option"><a href="#0">Settings</a></li>
+                    </ul>
+                </div>
+                <div class="log">
+                    <ul class="log-links">
+                        <li class="log-link"><a href="#0">Logged as user@email.com</a></li>
+                        <li class="log-link"><a href="#0">Sign out</a></li>
+                    </ul>
+                </div>
+            </nav>
+        </aside>
+        <main class="content">
+            <article class="artical-content">
+                <div class="artical-text">
+                    <h2 class="caption">Artical 1</h2>
+                    <p class="text">
+                        Lorem ipsum dolor sit, amet consectetur adipisicing elit. Quidem temporibus, accusantium pariatur, deleniti libero dolorum dignissimos quae unde accusamus autem ullam, recusandae culpa quas laudantium reiciendis. Voluptates voluptatem impedit exercitationem?
+                    </p>
+                </div>
+                <div class="btns">
+                    <button class="btn see-more">See more</button>
+                    <button class="btn like">Like</button>
+                    <button class="btn dislike">Dislike</button>
```

Comment:
When using `button` element don't forget to add `type="button"` attribute, since by default `button` has `type="submit"` which may lead to page reload if user tries to click it.